pkg/scene/delete.go

```diff
@@ -185,6 +189,13 @@ func (s *Service) deleteFiles(ctx context.Context, scene *models.Scene, fileDele
 	return nil
 }
 
+func DestroyFilter(ctx context.Context, scene *models.Scene, sceneFilter *models.SceneFilter, qb FilterDestroyer) error {
```

Comment:
I'm not sure I see the point of this - it's a barely-there wrapper around `qb.Destroy`.